src/js/components/Layer/StyledLayer.js

```diff
@@ -651,8 +742,15 @@ const desktopContainerStyle = css`
       props.theme,
       'right',
     )}px)`};
-  border-radius: ${props =>
-    props.plain ? 0 : props.theme.layer.border.radius};
+  ${props =>
+    props.plain || props.full === true
```

Comment:
Can we have `full` with `margin`? If so, then we should use `roundStyle()` in that case too.

Reply:
good call, let me look